kosat-core/src/commonMain/kotlin/org/kosat/CDCL.kt

```diff
@@ -68,10 +63,16 @@ class CDCL {
      */
     private val flpMaxProbes = 1000
 
+    /**
+     * Amount of [equivalentLiteralSubstitution] rounds before and after
+     * [failedLiteralProbing] to perform.
+     */
+    private val elsRounds = 5
+
     /**
      * The branching heuristic, used to choose the next decision variable.
      */
-    private val variableSelector: VariableSelector = VSIDS(numberOfVariables)
+    private val variableSelector: VariableSelector = VSIDS(assignment.numberOfVariables)
```

Comment:
Why pass numberOfVariables in VSIDS constructor? Anyway, it is 0 when the solver is created. Maybe simplify the interfaces and eliminate unnecessary functionality?

Reply:
Will do once I get to it. For now, it is not related to FLP, nor proof generation in FLP.